### PR TITLE
Removing cf-component-heading dependency for cf-component-modal::Moda…

### PR DESCRIPTION
**Pull request by jwineman**
`cf-component-page` and `cf-component-card` already don't have dependencies on `cf-component-heading`.  This PR removes the dependency from `<ModalTitle/>`

### CODE REVIEW

**Comment by jwineman** [comment]
@manatarms `<ModalTitle/>` === `<ModalHeading/>`

File: packages/cf-component-modal/src/ModalTitle.js [hunk 1]
```diff
@@ -1,15 +1,12 @@
...
 
 class ModalTitle extends React.Component {
   render() {
     return (
-      <div className="cf-modal__title">
-        <Heading size={3}>
-          {this.props.children}
-        </Heading>
-      </div>
+      <h3 className="cf-modal__title">
```
Comment (wyuenho) [inline packages/cf-component-modal/src/ModalTitle.js]: Use `<h1>` to be semantically correct.

**Comment by wyuenho** [comment]
Can we merge this?

**Comment by jwineman** [comment]
Yeah I'll rebase today.

**Comment by jwineman** [comment]
@tajo does my rebase of `package.json` look correct?

File: packages/cf-component-modal/package.json [hunk 2]
```diff
@@ -11,9 +11,9 @@
     "registry": "http://registry.npmjs.org/"
   },
   "dependencies": {
-    "cf-component-heading": "^4.3.1",
     "cf-component-icon": "^2.2.1",
     "cf-style-container": "^1.1.1",
+    "react-addons-css-transition-group": "^0.14.2 || ^15.0.0-0",
```
Comment (tajo) [inline packages/cf-component-modal/package.json]: use https://github.com/reactjs/react-transition-group instead 

https://facebook.github.io/react/blog/#discontinuing-support-for-react-addons

File: packages/cf-component-modal/src/ModalTitle.js [hunk 3]
```diff
@@ -1,16 +1,12 @@
-import React from 'react';
-import PropTypes from 'prop-types';
-import { Heading } from 'cf-component-heading';
+import React, { PropTypes } from 'react';
```
Comment (tajo) [inline packages/cf-component-modal/src/ModalTitle.js]: use `prop-types`

**Comment by jwineman** [comment]
I'm going to reopen this rebased off master.

**Pull request closed without merging**